src/python/gudhi/nerve_gic.pyx

```diff
@@ -56,17 +72,17 @@ cdef extern from "Nerve_gic_interface.h" namespace "Gudhi":
         void set_resolution_with_interval_length(double resolution)
         void set_resolution_with_interval_number(int resolution)
         void set_subsampling(double constant, double power)
-        void set_type(string type)
         void set_verbose(bool verbose)
         vector[int] subpopulation(int c)
-        void write_info()
-        void plot_DOT()
-        void plot_OFF()
+        double subcolor(int c)
+        void write_info(string data_name, string cover_name, string color_name)
+        void plot_DOT(string data_name)
+        void plot_OFF(string data_name)
         void set_point_cloud_from_range(vector[vector[double]] cloud)
         void set_distances_from_range(vector[vector[double]] distance_matrix)
 
 # CoverComplex python interface
-cdef class CoverComplex:
+cdef class GraphInducedComplex:
```

Comment:
In src/python/doc/nerve_gic_complex_ref.rst, you need to add `GraphInducedComplex` if you want the documentation to be available.

Reply:
Thanks! Done